besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -949,6 +960,17 @@ private void addConfigurationService() {
     }
   }
 
+  /* NOTE: Must be called after addConfigurationService */
+  private void addNodeKey() {
+    if (nodeKey == null) {
+      nodeKey = new NodeKey(nodeKeySecurityModuleProvider(nodeKeySecurityModuleProviderName));
+    }
+  }
+
+  public Optional<NodeKey> getNodeKey() {
```

Comment:
don't  _think_ this should expose the nodeKey - that should only be available from the BesuController.

Reply:
fixed.